CHANGELOG.md

```diff
@@ -1,9 +1,32 @@
-# dbt_linkedin_source v0.UPDATE.UPDATE
+# dbt_linkedin_source v0.8.0
+[PR #54](https://github.com/fivetran/dbt_linkedin_source/pull/54) includes the following updates:
 
- ## Under the Hood:
+## Breaking changes
```

Comment:
Changing the source name from `linkedin` to `linkedin_ads` is pretty huge. We should notify customers that the name has changed. That should ideally be the first bullet in this CHANGELOG.

Reply:
Updated.